src/store/cache/contentNode/hooks.ts

```diff
@@ -66,6 +66,12 @@ export const getFilteredNodes = ({
 
 // -------------------------------- Helpers  --------------------------------
 
+type HealthCheckResponseData = {
+  data: {
+    country: string
+  }
+}
+
 const processNode = async (node: Node, aud: Audius): Promise<ContentNode> => {
   const version = await Audius.getNodeVersion(node.endpoint)
```

Comment:
It looks like this call already hits `healthCheck` - maybe we can incorporate the country in that response to avoid another roundtrip?

Reply:
yay was hoping there was something like this!! will look into